packages/docs/components/sidebar/Sidebar.vue

```diff
@@ -1,64 +1,133 @@
-            v-for="(subMenuItem, index) in item.children"
-            :key="index"
-            :to="subMenuItem.name"
-            :title="$t(subMenuItem.displayName)"
-          />
-        </va-sidebar-link-group>
-        <va-sidebar-link
-          v-else
-          :key="key"
-          :minimized="minimized"
-          :active-by-default="item.name === route.name"
-          :to="item.name"
-        >
-          <span slot="title">{{ $t(item.displayName) }}</span>
-        </va-sidebar-link>
-      </template>
-    </template>
+          <template slot="header">
+            <va-list-item class="sidebar__category">
+              <va-list-item-section class="sidebar__category__section">
+                <va-list-item-label class="sidebar__category__label">
+                  {{ $t(route.displayName) }}
+                </va-list-item-label>
+              </va-list-item-section>
+              <va-list-item-section icon>
+                <va-icon
+                  :name="value[key] ? 'expand_less' : 'expand_more'"
+                  :color="value[key] ? 'primary' : ''"
+                />
+              </va-list-item-section>
+            </va-list-item>
+          </template>
+          <div v-for="(childRoute, index) in route.children" :key="index">
+            <va-list-label
+              v-if="childRoute.category"
+              class="sidebar__subcategory__label"
+              color="gray"
+            >
+              {{ $t(childRoute.category) }}
+            </va-list-label>
+            <va-list-item
+              :to="childRoute.name"
+              class="sidebar__link"
+              active-class="sidebar__link--active"
+            >
+              <va-list-item-section>
+                <va-list-item-label>
+                  {{ $t(childRoute.displayName) }}
+                </va-list-item-label>
+              </va-list-item-section>
+            </va-list-item>
+          </div>
+        </va-expand>
+      </va-expand-group>
+    </va-list>
   </va-sidebar>
 </template>
 
 <script lang="ts">
-// @ts-nocheck
-import { Component, Vue } from 'vue-property-decorator'
-import VaSidebar from './va-sidebar/VaSidebar.vue'
-import VaSidebarLink from './va-sidebar/VaSidebarLink.vue'
-import VaSidebarLinkGroup from './va-sidebar/VaSidebarLinkGroup.vue'
-import { navigationRoutes } from './NavigationRoutes.ts'
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { NavigationRoute } from './NavigationRoutes'
+import VaSearch from './va-sidebar/VaSearch.vue'
+
 @Component({
-  components: {
-    VaSidebarLinkGroup,
-    VaSidebarLink,
-    VaSidebar,
-  },
-  props: {
-    minimized: {
-      type: Boolean,
-      required: true,
-    },
-  },
+  components: { VaSearch },
 })
 export default class Sidebar extends Vue {
-  data () {
-    return {
-      items: navigationRoutes.routes,
+  value = [] as Array<boolean|undefined>
+
+  @Prop({ type: Array }) readonly navigationRoutes!: NavigationRoute[]
```

Comment:
Most likely you want to either mark it as required or define default. Right now sidebar will break if no navigation routes are provided.